nimbus_nlp/variable_extractor.py

```diff
@@ -77,7 +77,7 @@ def extract_variables(self, sent):
         request = self.get_prediction(sent)
 
         # Input had detected entities
-        if (request.payload != []):
+        if request.payload:
```

Comment:
I think @chidiewenike added this check to make sure there was something to index into - I think we need a way to check if the request.payload has items rather than just if it exists - its possible this is an empty object.